src/sniffer.c

```diff
@@ -6334,7 +6358,11 @@ static int RemoveFatalSession(IpInfo* ipInfo, TcpInfo* tcpInfo,
 {
     if (session && session->flags.fatalError == FATAL_ERROR_STATE) {
         RemoveSession(session, ipInfo, tcpInfo, 0);
-        SetError(FATAL_ERROR_STR, error, NULL, 0);
+        if(XSTRCMP("Unsupported TLS Version", error) != 0 &&
```

Comment:
I've considered removing this comparison and just having the last set error as the final error message for all cases.

Reply:
Do we really not have access to the real error number in the `session`? If not let's add it.